mpcs/src/sum_check/classic/coeff.rs

```diff
@@ -49,9 +49,7 @@ impl<E: ExtensionField> ClassicSumCheckRoundMessage<E> for Coefficients<E> {
     }
 
     fn sum(&self) -> E {
-        self[1..]
-            .iter()
-            .fold(self[0].double(), |acc, coeff| acc + coeff)
+        self[0] + self[..].iter().sum::<E>()
```

Comment:
Doubling the first element is a bit weird.  But the tests seem to expect it.  What's the reason for it?